services/core-web/common/actionCreators/noticeOfWorkActionCreator.js

```diff
@@ -378,12 +378,15 @@ export const deleteNoticeOfWorkApplicationReview = (applicationGuid, application
     .finally(() => dispatch(hideLoading()));
 };
 
-export const addDocumentToNoticeOfWork = (now_application_guid, payload) => (dispatch) => {
+export const editNoticeOfWorkDocument = (now_application_guid, mineDocumentGuid, payload) => (
+  dispatch
+) => {
   dispatch(showLoading("modal"));
   dispatch(request(reducerTypes.ADD_DOCUMENT_TO_NOTICE_OF_WORK));
   return CustomAxios()
     .put(
-      ENVIRONMENT.apiUrl + API.NOTICE_OF_WORK_DOCUMENT(now_application_guid),
+      `${ENVIRONMENT.apiUrl +
+        API.NOTICE_OF_WORK_DOCUMENT(now_application_guid)}/${mineDocumentGuid}`,
       payload,
```

Comment:
The const 'NOTICE_OF_WORK_DOCUMENT' should be updated to include the new param in the API.js instead.

```
export const NOTICE_OF_WORK_DOCUMENT = (now_document_guid, mineDocumentGuid) =>
  `/now-applications/${now_document_guid}/document/${mineDocumentGuid}`;
  ```

Reply:
this is used in multiple areas and not just for this. I dont see how this is bad or incorrect though.

Reply:
It's not necessarily bad, we just have a pattern for our API routes. If this is being used in multiple places that do not need `mineDocumentGuid` i guess we can keep it like this.